lib/active_record/typed_store/column.rb

```diff
@@ -1,23 +1,24 @@
 module ActiveRecord::TypedStore
-
-  class Column < ::ActiveRecord::ConnectionAdapters::Column
-    attr_reader :array, :blank
+  class Column
```

Comment:
Is this a column? I think it is more like an attribute since it doesn't match database columns.